odd-collector-sdk/odd_collector_sdk/secrets/aws/ssm_parameter_store.py

```diff
@@ -25,6 +32,37 @@ def __init__(self, **kwargs) -> None:
         )
         self._ssm_client = boto3.client("ssm", region_name=self._region_name)
 
+    @staticmethod
+    def _dynamically_get_aws_region(check_region_argument: str) -> str:
```

Comment:
What is `dynamically` stands for here? It seems just `_get_aws_region`

Reply:
Already changed in #35